docs/guide/install-and-run.asciidoc

```diff
@@ -0,0 +1,264 @@
+
+It will try to connect to Elasticsearch on localhost port 9200 and expose an API to agents on port 8200.
+You can change the defaults by supplying different addresses on the command line:
+
+[source,bash]
+----------------------------------
+./apm-server -e -E output.elasticsearch.hosts=ElasticsearchAddress:9200 -E apm-server.host=localhost:8200
+----------------------------------
+
+Or you can update the `apm-server.yml` configuration file:
+
+[source,yaml]
+----------------------------------
+apm-server:
+  host: localhost:8200
+
+output:
+  elasticsearch:
+    hosts: ElasticsearchAddress:9200
+----------------------------------
+
+NOTE: If you are using an X-Pack secured version of Elastic Stack,
+you need to specify credentials in the config file:
+
+[source,yaml]
+----
+output.elasticsearch:
+  hosts: ["ElasticsearchAddress:9200"]
+  username: "elastic"
+  password: "elastic"
+----
+
+
+
+[[secure-api-access]]
+[float]
+==== Secure access to the API
+If you change the listen address from `localhost` to something that is accessible from outside of the machine,
+we recommend setting up firewall rules to ensure that only your own systems can access the API.
+Alternatively,
+you can use a {apm-server-ref}/securing-apm-server.html[secret token and TLS].
+
+If you have APM Server running on the same host as your service, you can configure it to listen on a Unix domain socket.
+
+[[kibana-dashboards]]
+[float]
+==== Install the Kibana dashboards
+
+From APM Server and Kibana 6.4 on you can load dashboards directly via the {kibana-ref}/apm-getting-started.html[APM 
+Kibana UI].
+
+See an example screenshot of a Kibana dashboard:
+
+image::kibana-dashboard.png[Screenshot of a Kibana Dashboard]
+
+[[more-information]]
+[float]
+=== More information
+For detailed instructions on how to install and secure APM Server in your server environment,
+including details on how to run APM Server in a highly available environment,
+please see {apm-server-ref}/index.html[APM Server documentation].
+
+Once APM Server is up and running,
+you need to install an agent in your service.
+
+[[agents]]
+[float]
+=== Install and configure APM agents
+
+Agents are written in the same language as your service.
+Currently Elastic APM has agents for Node.js, Python, Ruby, Java, Go, and JavaScript.
+
+Setting up a new service to be monitored requires installing the agent,
+and configuring it with the address of your APM Server and the service name.
+
+[[choose-service-name]]
+[float]
+==== Choose a service name
+
+The service name is used by Elastic APM to differentiate between data coming from different services.
+
+Elastic APM includes the service name field on every document that it saves in Elasticsearch.
+If you change the service name after using Elastic APM,
+you will see the old service name and the new service name as two separate services.
+Make sure you choose a good service name before you get started.
+
+The service name can only contain alphanumeric characters,
```

Comment:
this is copied over, but (fyi) you don't have to break lines like this, feel free to change

Reply:
Haha, wish I would've known this was optional about three days ago :)